Dockerfile

```diff
@@ -51,6 +53,36 @@ USER 100
 
 ENTRYPOINT ["/usr/local/bin/dumb-init", "/usr/local/bin/consul-dataplane"]
 
+# FIPS release-default release image
+# -----------------------------------
+FROM gcr.io/distroless/base-debian11 AS release-fips-default
```

Comment:
Any specific reason to use debian here?  I think all of our container images are based on alpine which makes it easier to track and fix CVEs.

Reply:
This is an exact copy of the release-default target above, so if we want to change it there, we could then update this once that has been done

Reply:
Ahhh, I missed that. It should be ok!